docs/usage.md

```diff
@@ -3,130 +3,441 @@
+   - The assistant prompts you to provide the updated information for the record.
 
-Based on the reputation, the indicator reputation is updated to *Malicious*, if the reputation returned by VirusTotal is *Suspicious*; or the indicator reputation is updated to *Good* if the reputation is anything but *Suspicious*.
+2. **Provide Updated Information**:
+   - Specify the details to be updated in the record. For example:
 
-You may not be a seasoned playbook developer yet and may require some assistance with developing this playbook.
+    >*Update the alert status to "Resolved" and add a note: [Details of the note]*
 
-Use FortiAI to help with this task by clicking the **Playbook Generation** option in the FortiAI flyout.
+   - The assistant applies the changes to the record based on the provided information.
 
-Type your requirements, such as
+3. **Review and Confirm**:
+   - Review the updated details to ensure accuracy. Confirm the update to finalize the changes to the record.
 
->*Extract Indicators from the Alert Source Data and enrich them using VirusTotal. Based upon reputation, If `malicious` update the indicator reputation to `malicious` else if `suspicious` update the indicator reputation to `suspicious` else update indicator reputation to good*
+### Review Updated Records
 
-Click the **Post** icon. FortiAI, in response, designs the playbook and presents you with the proposed playbook outline:
+- Verify that the record has been updated correctly with the new information. Ensure that all changes are accurate and that the record reflects the most recent data.
 
-![FortiAI - Suggested playbook outline](../docs/res/FortiAIPBGeneration.png)
+### Integrate Updated Records into Workflow
 
-In the review mode itself, you can make modifications, such as changing the integration from VirusTotal to IPStack. Once you are satisfied, click **Ok, Generate Playbook**:
+- Utilize the updated records within your workflow to ensure that your data and analysis reflect the most current information. The updated records can aid in making informed decisions and managing tasks effectively.
 
-![FortiAI - Generate Playbook button](../docs/res/FortiAIPBGenerationReview.png)
+>[!TIP]
+>Regularly review and update records to ensure that all information remains current and accurate. This practice helps maintain the reliability of your data and supports effective decision-making.
 
-<table>
-    <th>NOTE</th>
-    <td>If any step uses a connector that has not been installed, then FortiAI will display a message informing the user that the specific step requires a connector that is not installed on the system.</td>
-</table>
+Updating records is a crucial task for maintaining accurate and relevant information within FortiSOAR. By following the outlined steps and using the FortiAI assistant’s capabilities, users can effectively manage and update records to support their workflow and data accuracy.
```

Comment:
this para is also talks the same thing about review

Reply:
Done